config/colcon.debian.upstream.yaml

```diff
@@ -20,6 +20,7 @@ filter_formula: "\
 (Package (% python3-colcon-installed-package-information), $Version (% 0.1.0*))|\
 (Package (% python3-colcon-lcov-result), $Version (% 0.5.0*))|\
 (Package (% python3-colcon-library-path), $Version (% 0.2.1*))|\
+(Package (% python3-colcon-meson), $Version (% 0.4.2*))|\
```

Comment:
Even though there is not technically a Debian release which includes this package I think it makes sense to keep the debian and ubuntu files in sync.